src/shared/components/pipeline-run-logs/logs/Logs.tsx

```diff
@@ -2,159 +2,176 @@ import * as React from 'react';
 import { useTranslation } from 'react-i18next';
 import { Alert } from '@patternfly/react-core';
 import { Base64 } from 'js-base64';
-import { useNamespace } from '~/shared/providers/Namespace';
 import { commonFetchText } from '../../../../k8s';
 import { getK8sResourceURL, getWebsocketSubProtocolAndPathPrefix } from '../../../../k8s/k8s-utils';
+import { MessageHandler, WebSocketOptions } from '../../../../k8s/web-socket/types';
 import { WebSocketFactory } from '../../../../k8s/web-socket/WebSocketFactory';
 import { PodModel } from '../../../../models/pod';
-import { ContainerSpec, PodKind } from '../../types';
-import { LOG_SOURCE_TERMINATED } from '../utils';
+import { useNamespace } from '../../../providers/Namespace';
+import { PodKind, ContainerSpec, ContainerStatus } from '../../types';
+import { containerToLogSourceStatus, LOG_SOURCE_TERMINATED } from '../utils';
+import LogViewer, { type Props as LogViewerProps } from './LogViewer';
 
-import './Logs.scss';
+type LogSources = { [containerName: string]: string };
+
+export const processLogs = (logSources: LogSources, containers: ContainerSpec[]): string => {
+  let allLogs = '';
+  // ANSI escape codes for text styling
+  const BOLD = '\x1b[1m';
+  const WHITE_BACKGROUND_COLOR = '\x1b[47m';
+  const BLACK_TEXT_COLOR = '\x1b[30m';
+  const RESET = '\x1b[0m';
```

Comment:
I added this styling just to make the container names more visible :) 

that's how it looks like:

![Screenshot 2025-08-01 at 16 40 51](https://github.com/user-attachments/assets/16c9d331-3470-4731-a5f5-887de0b3395c)

is it okay to leave it like this? or should I remove the styling?

cc: @sahil143 @pankajshivpuje